Source/WebCore/platform/glib/PasteboardGLib.cpp

```diff
@@ -266,6 +266,29 @@ void Pasteboard::setDragImage(DragImage, const IntPoint&)
 }
 #endif
 
+static std::array<ASCIILiteral, 6> imageTypes()
+{
+    static std::array<ASCIILiteral, 6> types = { "image/png"_s, "image/jpeg"_s, "image/gif"_s, "image/bmp"_s, "image/vnd.microsoft.icon"_s, "image/x-icon"_s };
+    return types;
+}
```

Comment:
This does not need to be a function, it can be a constant global:


```suggestion
static constexpr std::array<ASCIILiteral, 6> imageTypes = { "image/png"_s, "image/jpeg"_s, "image/gif"_s, "image/bmp"_s, "image/vnd.microsoft.icon"_s, "image/x-icon"_s };
```

Reply:
Done.